Add `FileTypeFilter` plugin (#412)
gardener/docforge: docs/plugin/filetypefilter.md

# Using `File type filter`

> Docforge can construct the content bundle from multiple repository sources and every one could have certain limitations for file types.
> To make it work, use the flag `content-files-formats` and set the extensions of the files allowed for Docforge to process.

Reviewer: ```suggestion
Docforge can construct the content bundle from multiple repository sources each containing various types of files. 
To limit what the bundle can contain, use the flag `content-files-formats` and set the extensions of the files allowed for Docforge to process.
```
